app/Makefile

```diff
@@ -37,13 +36,32 @@ release-build:
 ##################################################
 # Local development
 ##################################################
-dev: # Run the Next.js local dev server in Docker
+dev: pre-dev # Run the Next.js local dev server in Docker
 	docker compose up --detach nextjs
 	docker compose logs --follow nextjs
 
-storybook: # Run the Storybook local dev server in Docker
+storybook: pre-dev # Run the Storybook local dev server in Docker
 	docker compose up --detach storybook
 	docker compose logs --follow storybook
 
 stop:
 	docker compose down
+
+pre-dev:
+	@# Check for existing "next-dev" container and remove it if it exists
```

Comment:
Just for my learning, why do we want to remove the containers each time?

Reply:
I also initially thought that NODE_ENV=development was set by `next dev`. This isn't a requirement to get hot-reload working within the container. It's just something I stumbled across that "could've" been the culprit and was thinking that it should be set to development either way.

Reply:
> I also initially thought that NODE_ENV=development was set by next dev. This isn't a requirement to get hot-reload working within the container. It's just something I stumbled across that "could've" been the culprit and was thinking that it should be set to development either way.

If you have this running already, can you add a log somewhere to see what it outputs for `NODE_ENV` without this? My impression is it's already `development` and wasn't the culprit. I've only ever seen `NODE_ENV=production` getting explicitly set in other projects.